### ISSUE TITLE
[Not-an-issue] Donations

### ISSUE THREAD

**Issue by AshourAziz** [Borlea/EchoPet#57]
You've allowed us to bring back pets as a cosmetic donation & in-game token purchases.
This has been an amazing thing, allowing us to easily re introduce pets after a month hideous

Although we haven't gotten much donations for pets, We'd like to make a donation for supporting such a larger project and a large update :-)

So if we could get some sort of paypal email or a paypal.me link that would be great :-)

http://i.imgur.com/b1IoZVW.png

Tl;Dr http://i.imgur.com/b1IoZVW.png how 2 donate?

**Comment by Borlea** [comment]
No donations will be accepted.
